client/web/compose/src/views/Public/Pages/Records/View.vue

```diff
@@ -245,6 +274,23 @@ export default {
         this.$router.push({ name: 'page.record.edit', params: this.$route.params })
       }
     },
+
+    handleRedirectToPrevOrNext (index) {
```

Comment:
i would accept 'prev'/'next' param. And just use the getter from the store to get the apropriate ID.